libcob/fbdb.c

```diff
@@ -1418,9 +1424,14 @@ ix_bdb_open (cob_file_api *a, cob_file *f, char *filename, const enum cob_open_m
 				if (f->keys[i].tf_duplicates) {
 					p->db[i]->set_flags (p->db[i], DB_DUP);
 				}
+				/* TODO: add national compare function later */
```

Comment:
this bunch does not include a merge to TODO, so I guess that's missing - please review the "NATIONAL support" part there and add this as a sub-item (not necessary for the first 4.x release, but something we want to inspect someday and can easily forget)

Reply:
Well, there is nothing about NATIONAL support in the TODO file. Guess I'll have to add this.

Reply:
Agreed - in 3.x (maybe start that locally already) after we merged that bunch to trunk. I'm looking forward to the "merge change" parts of this bunch, which will mostly if not all be about the backward compat stuff.

Reply:
resolved by ongoing #219